test/specs/buttons.js

```diff
@@ -0,0 +1,27 @@
+const crossHomePage = require('./crossHomePage'); //importing Page Objects
+
+describe('CrossfadeYT test cases for adding and removing songs', function(){
+  it('should add a song when clicking on add song button', function(){
+    const crossfadeYT = new crossHomePage();
+
+    crossfadeYT.open(); //open crossfadeyt home page
+    crossfadeYT.addSong('rYEDA3JcQqw', '10', '100'); //add a song
+
+    const addedSong = $('.song-detail');
+    expect(addedSong).toBeDisplayed(); //check if a song is added
```

Comment:
We should check all the details of the song to make sure it's correct: index, ID, start, end time